util/admin/admin.go

```diff
@@ -534,7 +528,7 @@ func CompareTableRecord(sessCtx sessionctx.Context, txn kv.Transaction, t table.
 			return true, nil
 		}
 
-		if !reflect.DeepEqual(vals, vals2) {
+		if !compareDatumSlice(sc, vals, vals2) {
```

Comment:
Why not use `sessCtx.GetSessionVars().StmtCtx` here?

Reply:
done.